### PR TITLE
Documentation - Manual testing: Resizing

### PR DESCRIPTION
**Pull request by Wei-Xinn**
This PR adds resizing for manual testing.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1272,3 +1272,27 @@ testers are expected to do more *exploratory* testing.
...
+   Expected: The Help Window's dimesions will not go above `1250x700`.
+
+### Resizing the Onboarding Guide prompt
+
+1. Resizing the Onboarding Guide prompt
+
+   1. Prerequisite: Onboarding Guide prompt is open
+
+   1. Test case: Resize Onboarding Guide prompt to be smaller <br>
+   Expected: The prompt's dimesions will not go below `150x500`.
```
Comment (KwanHW) [inline docs/DeveloperGuide.md:1295]: Should be `500x150` (width x height). 

Also, I think the expected result can be written along the lines of:
```suggestion
   Expected: The prompt's dimensions will not go have width smaller than 500 or height smaller than 150.
```
If anyone have a better proposal for the phrasing feel free to suggest!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1272,3 +1272,27 @@ testers are expected to do more *exploratory* testing.
...
+
+   1. Test case: Resize Onboarding Guide prompt to be larger <br>
```
Comment (KwanHW) [inline docs/DeveloperGuide.md:1297]: Same comment as above. Check your dimensions and (if needed) maybe have more explicit phrasing?

**Review by KwanHW — COMMENTED** [review]
I have a few suggestions to improve on the test cases

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Wei-Xinn) [reply docs/DeveloperGuide.md]: Alright changed all to width x height. Would like to change to explicit bt dk if there is a unit width and height. like 500px or smth

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1280,10 +1280,10 @@ testers are expected to do more *exploratory* testing.
    1. Prerequisite: Help Window is open
 
    1. Test case: Resize Help Window to be smaller <br>
-   Expected: The Help Window's dimesions will not go below `1000x600`.
+   Expected: The Help Window's dimensions will not go below `600x1000`.
 
    1. Test case: Resize Help Window to be larger <br>
-   Expected: The Help Window's dimesions will not go above `1250x700`.
+   Expected: The Help Window's dimensions will not go above `700x1250`.
```
Comment (dandaandaaaaaan) [inline docs/DeveloperGuide.md:1286]: your previous dimensions for help window were already in width x height, u flipped it again.

prompt one is correct

**Review by dandaandaaaaaan — CHANGES_REQUESTED** [review]
the dimensions u accidentally flipped for one part

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Wei-Xinn) [reply docs/DeveloperGuide.md]: Fixed

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Pull request merged**